### PR TITLE
Some (small) problems I came across

### PR DESCRIPTION
**Pull request by omrihar**
Thanks for the great article! It's a really nice explanation of the Metropolis-Hastings algorithm and of how to do sampling in Computational Bayesian Statistics! I came across some small (but important) problems in the code. 
Most notably - when there is more than one data point, the likelihood function needs to be the product of the likelihoods of each individual data point - which I added above to the code for the functions. Perhaps you would want to mention this differently in the text.
Another problem is that you need to sample the unbounded sigma value from the previous *unbounded* sigma value, rather than the bounded one.
A third small correction - the pdf in the text above stands for density rather than distribution, as your title suggests, but the text itself used the word distribution, which is incorrect in this context.

I really enjoyed your article and hope my small proposed changes will help improve it a bit further :)

Omri

### CODE REVIEW

File: docs/machine-learning/computational-bayesian-stats.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ class Normal:
 Now, I also stated that the normal distribution has a math function
 that we can use to allocate credibility points to the number line.
 This function also has a name,
-called a "probability distribution function", or the "PDF".
+called a "probability density function", or the "PDF".
```
Comment (ericmjl) [inline docs/machine-learning/computational-bayesian-stats.md:56]: Ah yes, I saw this one, but didn't find the bandwidth to correct it in time. Thank you!

File: docs/machine-learning/computational-bayesian-stats.md [hunk 2]
```diff
@@ -538,7 +538,7 @@ sigma_prev_unbounded = np.random.normal(0, 1)
 for i in range(1000):
     # ...
     # Propose in unconstrained space
-    sigma_t_unbounded = np.random.normal(sigma_prev, 0.1)
+    sigma_t_unbounded = np.random.normal(sigma_prev_unbounded, 0.1)
```
Comment (ericmjl) [inline docs/machine-learning/computational-bayesian-stats.md:541]: Good catch!

**Review by ericmjl — APPROVED** [review]

**Comment by ericmjl** [comment]
Thanks @omrihar!

**Comment by omrihar** [comment]
You're welcome!

**Pull request merged**